website/static/js/pages/project-settings-page.js

```diff
@@ -227,7 +227,13 @@ $(document).ready(function() {
       var unchecked = checkedOnLoad.filter('#selectAddonsForm input:not(:checked)');
 
       if(unchecked.length > 0 || checked.length > 0) {
-        return 'The changes on addon setting are not submitted!';
+          return 'The changes on addon setting are not submitted!';
+      }
+    /* Before closing the page, Check whether changes made to category, title or description are updated or not */
+      if ($('#titleID')[0].value !== (projectSettingsVM).titlePlaceholder  ||
```

Comment:
Drive-by comment: Rather than relying on CSS IDs, which are brittle (i.e. the code will break if the IDs are changed or removed), can you use instead access attributes/observables on the view model? For example: `projectSettingsVM.title()` rather than `$('#titleID')[0].value`